src/main/java/org/folio/rest/impl/ItemStorageAPI.java

```diff
@@ -56,6 +56,13 @@ public void postItemStorageItems(
       RoutingContext routingContext, Map<String, String> okapiHeaders,
       Handler<AsyncResult<Response>> asyncResultHandler,
       Context vertxContext) {
+    //if the initial status is "Available", set the last updated date of that 
```

Comment:
It is also possible to create an item using Item batch API, could you please apply the same logic to the `org.folio.rest.impl.ItemBatchSyncAPI` class?

Reply:
Good catch, done.

Reply:
Hmm.  Trying to set the updated date within the class results in what looks like every test that batch-creates items failing with a 400 error.  It looks like implementing that in ItemBatchSyncAPI is not going to be as simple as adding the same logic.  I'll probably ned to close this PR and go figure out how to implement it, and then re-issue.

Reply:
Also, from looking at the code, it looks like that class may actually be handing off the items one by one to the ItemStorageAPI class, in which case the status date should get set by the post method there.  I'll need to run a batch load to be sure.

Reply:
Okay, I've written a test, and assuming it's correct, it's indicating that batch-created items don't have status dates set with present changes.  So I'll have to go figure out how to make that happen.